src/utils/update/mapper.utils.ts

```diff
@@ -24,14 +24,11 @@ export const mapSelfLink = (selfLink: string | undefined): string | undefined =>
   return path.length > 0 ? path[path.length - 1] : undefined;
 };
 
-export const mapInputDate = (date: string | undefined): InputDate => {
+export const mapInputDate = (date: string | undefined): InputDate | undefined => {
```

Comment:
Is there a reason from the method caller why this takes undefined or return undefined type?

Reply:
line 29 now returns undefined. Since function used by date of registration/creation I've asked that to be tested.